File: src/point2.py
import point1

class Sentence:
    _data = ""

    def __init__(self):
        self

    def concat(self, data):
        self._data += data
    
    def __str__(self):
        return self._data

codeTimer = point1.CodeTimer("Point 2")
codeTimer.start()

sentence = Sentence()

for user in point1.data.users:
    for word in user['description']:
        sentence.concat(word)

codeTimer.stop()
print(f'{codeTimer.result()}')
print(len(sentence.__str__()))

File: src/point2.php
<?php
require_once 'point1.php';

class Sentence
{
    private string $data = '';

    public function concat(string $data) : void
    {
        $this->data .= $data;
    }

    public function __toString()
    {
        return $this->data;
    }
}

$codeTimer = new CodeTimer('Point 2');
$codeTimer->start();
$sentence = new Sentence;

foreach ($data->users as $user) {
    foreach($user->description as $word) {
        $sentence->concat($word);
    }
}
$codeTimer->stop();
echo $codeTimer->result() . "\n";

var_dump(strlen($sentence));
